api/pkg/db/model/model.go

```diff
@@ -7,33 +7,33 @@ import (
 type (
 	Category struct {
 		gorm.Model
-		Name string `gorm:"size:100;not null;unique"`
+		Name string `gorm:"not null"`
 		Tags []Tag
 	}
 
 	Tag struct {
 		gorm.Model
-		Name       string `gorm:"size:100;not null;unique"`
+		Name       string `gorm:"not null"`
 		Category   Category
 		CategoryID int
 		Resources  []*Resource `gorm:"many2many:resource_tags;"`
 	}
 
 	Catalog struct {
 		gorm.Model
-		Name       string
-		Type       string
-		URL        string
-		Owner      string
+		Name       string `gorm:"not null"`
```

Comment:
`catalog - name - type` combination will be unique if you can add a constraint for that, would be nice